### PR TITLE
add email_verified: true

### PR DESCRIPTION
**Pull request by taoliu12**
added "email_verified: true," to both seed users, and Login works correctly now.

### CODE REVIEW

**Review by esteban-gs — CHANGES_REQUESTED** [review]
Nice! this should get us closer to closing this issue. A couple of changes in comments and here:

- At one point we added the `email_verified` property back to the `create-user-dto.ts` class. This should be reverted. Even though the card doesn't require if explicitly, we could benefit from fixing this. 
- Requested changes:
  - Remove the `email_verified?` property from `create-user.dto` - `src/acccount-manager/dto/create-user.dto.ts`
  - In the seed date. use the `CreateUserInternal` - `src/acccount-manager/dto/create-user.internal.ts` type instead of `CreateUserDto`. 

The main idea behind this is to disallow the controller's ability to update the `email_verified` field, since it controls access to the app. Internally, we can still override it by using the service type `CreateUserInternal` which has access to that property.

File: server/src/acccount-manager/dto/create-user.dto.ts
Comment (taoliu12) [inline server/src/acccount-manager/dto/create-user.dto.ts:1]: Sorry, not sure why the entire file is diff'ed.
But I removed the 
`  @IsBoolean()
  email_verified?: boolean;`

**Comment by taoliu12** [comment]
@esteban-gs I made the requested changes, and re-seeded to make sure login still works.  

Also, sorry, not sure why the entire file is diff'ed .
But I removed the email_verified? property from create-user.dto

**Review by esteban-gs — APPROVED** [review]
Looks good to me!

**Pull request merged**